fixtures/purges/purge.yaml

```diff
@@ -9,7 +9,7 @@ interactions:
       - d631487398e146a8593ddd71d384c7e6
       User-Agent:
       - FastlyGo/0.2 (+github.com/sethvargo/go-fastly; go1.6.2)
-    url: https://api.fastly.com/https://releases.hashicorp.com
+    url: https://api.fastly.com/https:/releases.hashicorp.com
```

Comment:
This is incorrect. The url should have two slashes in the URL, since it's the full URL to purge.